### DISCUSSION TITLE
Error message with Create Connection from IOM

### DISCUSSION THREAD

**Discussion by s4lmonsushi** [idea-statica/ideastatica-public#524] [Rhino/Grasshopper] [answered]
Hi,

I am getting an error message as below:

1. Solution exception:Error 'The server was unable to process the request due to an internal error.  For more information about the error, either turn on IncludeExceptionDetailInFaults (either from ServiceBehaviorAttribute or from the <serviceDebug> configuration behavior) on the server in order to send the exception information back to the client, or turn on tracing as per the Microsoft .NET Framework SDK documentation and inspect the server trace logs.'

with the create connection component. I am using the example file "Create Connection using IOM Components". What is likely causing this issue and how can I resolve it?
I have enterprise license for IdeaStatica.

**Comment by nathanlukeIDEA** [comment, marked as answer]
HI @s4lmonsushi, which version of IDEA StatiCa are you using? we had an issue with the 24.0.0 release which has been fixed in the 24.0.1 release.

**Reply by s4lmonsushi** [reply]
Hi Nathan, thanks it was 24.0.0. Updating to 24.0.1 has fixed the problem.
